src/components/Header.tsx

```diff
@@ -75,6 +75,22 @@ export const Header: React.FunctionComponent<HeaderProps> = (
 
   const Title = isHome ? "h1" : "div";
 
+  const Logo = (
```

Comment:
```suggestion
  const logo = (
```

TitleCase implies that it's a class or component, not an instance.